lists2safebrowsing.py

```diff
@@ -664,7 +678,7 @@ def start_versioning(config, chunknum, shavar_prod_lists_branches):
 
 def main():
     config = configparser.ConfigParser()
-    filename = config.read(["shavar_list_creation.ini"])
+    filename = config.read(["shavar_list_creat_latest_prod.ini"])
```

Comment:
Is this intentional?

Reply:
Corrected [453dcea](https://github.com/mozilla-services/shavar-list-creation/pull/176/commits/453dcea4ea3dbbf37f82a5f3458ac50699cef302)